package.json

```diff
@@ -69,7 +69,12 @@
     "@sideway/formula": "^3.0.1",
     "@types/react": "17.0.47",
     "**/@angular-devkit/build-angular/minimatch": "3.0.5",
+    "**/@angular-devkit/build-angular/semver": "7.5.4",
+    "**/@angular/cli/semver": "7.5.4",
     "**/@angular-devkit/build-angular/webpack": "^5.76.0",
+    "**/@changesets/cli/**/semver": "7.5.4",
```

Comment:
Can we bump changeset directly instead?

Reply:
We recently bumped changesets but some of it's dependent packages still have this issue it seems - https://github.com/aws-amplify/amplify-ui/pull/4210

Reply:
Yeah it is from `normalize-package-data`, I can change the scope to only be to that package
```
@changesets/cli@2.26.2 requires semver@^7.5.3 via a transitive dependency on @changesets/get-dependents-graph@1.3.6
@changesets/cli@2.26.2 requires semver@^7.5.3 via a transitive dependency on @changesets/apply-release-plan@6.1.4
@changesets/cli@2.26.2 requires semver@^7.5.3 via a transitive dependency on @changesets/assemble-release-plan@5.2.4
@changesets/cli@2.26.2 requires semver@2 || 3 || 4 || 5 via a transitive dependency on normalize-package-data@2.5.0
@changesets/cli@2.26.2 requires semver@^7.5.3
```